### PR TITLE
Added albanian & korean to the installer

### LINKED ISSUE #15438: Add Albanian and Korean to the installer
**Is your feature request related to a problem?**
It is not a bug, but a default parameter to update.

**Describe the solution you'd like**
For the 1.7.7 version, following recent activity on Crowdin, it would be nice to add Albanian and Korean to the PrestaShop installer.

**Why?**
Since the installer helps you installing PrestaShop, it must be accessible to as many people as possible. And this is why it is available in many languages. If a language has a high level of translation on Crowdin, which is the case for both Albanian and Korean projects, without being available in the installer, we should consider adding it.

**How?**
Here, a pull request must be created to add a new folder in _install-dev/langs_, named with the code of the language we want to add.

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Added albanian & korean to the installer
| Type?         | improvement
| Category?     | LO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #15438
| How to test?  | * Install PrestaShop in Shqip (Albanian) & the installer must be in Shqip (Albanian)<br> * Install PrestaShop in 한국어 (Korean) & the installer must be in 한국어 (Korean)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16416)

### CODE REVIEW

File: install-dev/langs/ko/language.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<language>
+	<name><![CDATA[한국어 (Korean)]]></name>
+	<locale>ko-KR</locale>
+	<language_code>ko</language_code>
+	<date_format_lite>Y-m-d.</date_format_lite>
```
Comment (LouiseBonnard) [inline install-dev/langs/ko/language.xml]:
```suggestion
	<date_format_lite>Y-m-d</date_format_lite>
```

**Review by LouiseBonnard — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]
No reason to delete the `var/cache/.gitkeep`

**Comment by Progi1984** [comment]
@PierreRambaud Wow ! I just added it... I don't know why it was removed.

**Review by LouiseBonnard — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @Progi1984 and @LouiseBonnard

**Pull request merged**